client/asset/eth/nodeclient.go

```diff
@@ -500,6 +500,32 @@ func (n *nodeClient) isRedeemable(secretHash [32]byte, secret [32]byte, contract
 	})
 }
 
+// transactionConfirmations gets the number of confirmations for the specified
+// transaction.
+func (n *nodeClient) transactionConfirmations(ctx context.Context, txHash common.Hash) (uint32, error) {
+	// We'll check the local tx pool first, since from what I can tell, a light
+	// client always requests tx data from the network for anything else.
```

Comment:
Curious what you discovered regarding network requests.

I started digging and it looks like it at least has a local db that it checks before sending off a request, like a cache.
From les/api_backend.go, `(*LesApiBackend).GetTransaction` > `light.GetTransaction`, then in light/odr_util.go:
`GetTransaction` > `GetBody` > `GetBodyRLP` > `rawdb.ReadBodyRLP` otherwise `odr.Retrieve` for the data.